docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-php.md

```diff
@@ -0,0 +1,163 @@
+    - **SSL_ROOT_CERT** - the full path to the Yugabyte Cloud cluster CA certificate.
+
+3. Save the file.
+
+## Run the application
+
+Run the application.
+
+```sh
+$ php sample-app.php
+```
+
+You should see output similar to the following:
+
+```output
+>>>> Successfully connected to YugabyteDB!
+>>>> Successfully created table DemoAccount.
+>>>> Selecting accounts:
+name = Jessica, age = 28, country = USA, balance = 10000
+name = John, age = 28, country = Canada, balance = 9000
+>>>> Transferred 800 between accounts.
+>>>> Selecting accounts:
+name = Jessica, age = 28, country = USA, balance = 9200
+name = John, age = 28, country = Canada, balance = 9800
+```
+
+You have successfully executed a basic PHP application that works with Yugabyte Cloud.
+
+## Explore the application logic
+
+Open the `sample-app.php` file in the `yugabyte-simple-php-app` folder to review the methods.
+
+### connect
+
+The `connect` method establishes a connection with your cluster via the php-pgsql driver.
+
+```php
+$conn = new PDO('pgsql:host=' . HOST . ';port=' . PORT . ';dbname=' . DB_NAME .
+                ';sslmode=' . SSL_MODE . ';sslrootcert=' . SSL_ROOT_CERT,
+                USER, PASSWORD,
+                array(PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION,
+                        PDO::ATTR_EMULATE_PREPARES => true,
+                        PDO::ATTR_PERSISTENT => true));
+```
+
+### create_database
+
+The `create_database` method uses PostgreSQL-compliant DDL commands to create a sample database.
+
+```php
+$conn->exec('DROP TABLE IF EXISTS DemoAccount');
+
+$conn->exec('CREATE TABLE DemoAccount ( 
+                id int PRIMARY KEY, 
+                name varchar, 
+                age int, 
+                country varchar, 
+                balance int)');
+
+$conn->exec("INSERT INTO DemoAccount VALUES 
+                (1, 'Jessica', 28, 'USA', 10000), 
+                (2, 'John', 28, 'Canada', 9000)");
+```
+
+### select_accounts
+
+The `select_accounts` method queries your distributed data using the SQL `SELECT` statement.
+
+```php
+$query = 'SELECT name, age, country, balance FROM DemoAccount';
+
+foreach ($conn->query($query) as $row) {
+    print 'name=' . $row['name'] . ', age=' . $row['age'] . ', country=' . $row['country'] . ', balance=' . $row['balance'] . "\n";
+}
+```
+
+### transfer_money_between_accounts
+
+The `transfer_money_between_accounts` method updates your data consistently with distributed transactions.
+
+```php
+try {
+    $conn->beginTransaction();
+
+    $conn->exec("UPDATE DemoAccount SET balance = balance - " . $amount . " WHERE name = 'Jessica'");
+    
+    $conn->exec("UPDATE DemoAccount SET balance = balance + " . $amount . " WHERE name = 'John'");
+
+    $conn->commit();
+
+    print ">>>> Transferred " . $amount . " between accounts\n";
+
+} catch (PDOException $e) {
```

Comment:
The double-spacing just seems odd. It's in the source code too, but it's weird there too.
```suggestion
try {
    $conn->beginTransaction();
    $conn->exec("UPDATE DemoAccount SET balance = balance - " . $amount . " WHERE name = 'Jessica'");
    $conn->exec("UPDATE DemoAccount SET balance = balance + " . $amount . " WHERE name = 'John'");
    $conn->commit();
    print ">>>> Transferred " . $amount . " between accounts\n";
} catch (PDOException $e) {
```